docs/en/Concept.md

```diff
@@ -0,0 +1,41 @@
+# Basic Concept                                
+##
+## 1 Message Model
+
+The RocketMQ message model is mainly composed of a Producer, a Broker and a Consumer.The Producer is responsible for producing messages, the Consumer is responsible for consuming messages, and the Broker is responsible for storing messages.     
+A Broker corresponds to one server during actual deployment, and each Broker can store messages from multiple topics, and messages from each Topic can be stored in a different Broker by sharding strategy.     
+A Message Queue is used to store the physical addresses of messages, and the Message addresses in each Topic are stored in multiple Message queues.The Consumer Group consists of multiple Consumer instances.    
+##  2 Producer
+ A producer is responsible for producing messages, typically by business systems.It sends messages generated by the business application systems to brokers. RocketMQ provides multiple paradigms of sending: synchronous, asynchronous,sequential and one-way.Both synchronous and asynchronous methods require the Broker to return confirmation information, while one-way sending is not required.
+## 3 Consumer
+ A Consumer responsible for consuming messages, typically the background system responsible for asynchronous consumption.A Consumer pulls messages from brokers and feeds them into application. In perspective of user application, two types of consumers are provided:Pull Consumer and Push Consumer.
+## 4 Topic
+  Topic refers to a collection of messages, each of which contains several messages, each of which can belong to only one topic, and is the basic unit of RocketMQ for message subscription.
```

Comment:
1. I recommend you to split it into several simple sentences to make it easier to express and understand.
Something like this: 
The Topic refers to a collection of one kind message. Each topic contains several messages and one message can only belong to one topic. The Topic is the basic unit of RocketMQ for message subscription.